modules/configure_project/outputs.tf

```diff
@@ -18,7 +18,7 @@ output "watsonx_project_url" {
   description = "The URL of the watsonx project that's created."
```

Comment:
Can you please change this to `The URL of the watsonx project that is created.` ?

Reply:
Done.